src/test/java/api/support/matchers/EventMatchers.java

```diff
@@ -0,0 +1,125 @@
+package api.support.matchers;
+
+import static com.jayway.jsonpath.matchers.JsonPathMatchers.hasJsonPath;
+import static org.folio.circulation.support.JsonPropertyFetcher.getBooleanProperty;
+import static org.hamcrest.core.Is.is;
+
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeDiagnosingMatcher;
+
+import io.vertx.core.json.JsonObject;
+
+public class EventMatchers {
+
+  public static Matcher<JsonObject> isValidItemCheckedOutEvent(JsonObject loan) {
+    return JsonObjectMatcher.allOfPaths(
+      hasJsonPath("eventType", is("ITEM_CHECKED_OUT")),
```

Comment:
Can the `EventTypeMatchers.isItemCheckedOutEventType` be reused?

Reply:
Refactored